ui/app/templates/components/generated-item-list.hbs

```diff
@@ -0,0 +1,81 @@
+<PageHeader as |p|>
+  <p.top>
+    {{#key-value-header path="vault.cluster.access.methods"}}
+      <li>
+        <span class="sep">
+          /
+        </span>
+        {{#link-to "vault.cluster.access.methods"}}
+          methods
+        {{/link-to}}
+      </li>
+    {{/key-value-header}}
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      {{capitalize method}}
```

Comment:
I think I asked about this before - but do we want the method here or the id - and I don’t think we want to capitalize it - I think `ldap` is better than `Ldap`.

Reply:
`method` is the `id` here. naming is weird. I can take out the capitalization though, I agree. I just based it on how we handle this everywhere else